onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -24,6 +24,7 @@
 '|^test_cast_STRING_to_FLOAT_cpu.*'
 '|^test_convtranspose_1d_cpu.*'
 '|^test_convtranspose_3d_cpu.*'
+'|^test_constantofshape_*.*'
```

Comment:
ConstantOfShape op is supported by the runtime, but the node tests don't comply to the op spec. Bug opened - https://github.com/onnx/onnx/issues/1842